EIPS/eip-1052.md

```diff
@@ -24,7 +24,7 @@ takes one argument from the stack, zeros the first 96 bits
 and pushes to the stack the keccak256 hash of the code of the account 
 at the address being the remaining 160 bits. 
 
-In case the account does not exist `0` is pushed to the stack.
+In case the account does not exist or is empty `0` is pushed to the stack.
```

Comment:
Did you mean it's bytecode empty?

```suggestion
In case the account does not exist or the bytecode is empty `0` is pushed to the stack.
```

Reply:
We’re using EIP-161’s definition of “empty” here — having no nonce, balance or code.

Reply:
Ah, I see!

Reply:
Shouldn't it note as a comment that EIP-161's definition is used?

Reply:
Yes, I'd like to have this comment as well. Something like 

> or is empty (as defined by EIP-161)